felix.go

```diff
@@ -398,6 +398,13 @@ configRetry:
 	if configParams.PrometheusMetricsEnabled {
 		log.Info("Prometheus metrics enabled.  Starting server.")
 		go servePrometheusMetrics(configParams)
+		gaugeHost := prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Please move this above the `go servePrometheusMetrics()` so that there's no race condition where we server metrics without this one being defined.